docs/wpf/getting-started.md

```diff
@@ -1 +1,299 @@
+
+```cs
+using Prism.Commands;
+using Prism.Mvvm;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WpfApp1.ViewModels
+{
+    public class MainWindowViewModel : BindableBase
+    {
+        private Services.ICustomerStore _customerStore = null;
+
+        public MainWindowViewModel(Services.ICustomerStore customerStore)
+        {
+            _customerStore = customerStore;
+        }
+
+
+        public ObservableCollection<string> Customers { get; private set; } =
+            new ObservableCollection<string>();
+
+
+        private string _selectedCustomer = null;
+        public string SelectedCustomer
+        {
+            get => _selectedCustomer;
+            set
+            {
+                if (SetProperty<string>(ref _selectedCustomer, value))
+                {
+                    Debug.WriteLine(_selectedCustomer ?? "no customer selected");
+                }
+            }
+        }
+
+        private DelegateCommand _commandLoad = null;
+        public DelegateCommand CommandLoad =>
+            _commandLoad ?? (_commandLoad = new DelegateCommand(CommandLoadExecute));
+
+        private void CommandLoadExecute()
+        {
+            Customers.Clear();
+            List<string> list = _customerStore.GetAll();
+            foreach (string item in list)
+                Customers.Add(item);
+        }
+    }
+}
+```
+
+A bit of an explanation on what is happening here. MainWindowViewModel has a dependency on the ```ICustomerStore``` interface, so that interface has to be registered in the ```App.RegisterTypes``` so that it's implementation can be handled by the dependency container. There is a ```Customers``` property that is bound to the listview in the user interface and a ```SelectedCustomer``` that is bound to the currently selected item in the list view.
+
+There is also CommandLoad object that implements the ```ICommand``` interface. This has an ```Execute``` method that is called when the user clicks on the button. Prism implements the ```ICommand``` interface with ```DelegateCommand``` class that allows delegates to be passed in to handle implementing the ```ICommand``` interface. In the case of ```CommandLoad```, the ```CommandLoadExecute``` function is passed in as the delegate and now, whenever the WPF binding system tries to execute ```ICommand.Execute```, ```CommandLoadExecute``` is invoked.
+
+For more details on DelegateCommand, see [Commanding](../commanding.md).
+
+Now there is a view and a view model, but how are they linked together? There are a couple of different ways this can be done, one explicitly and one via convention.
+
+### Explicit
```

Comment:
I would remove this part. Not needed for a getting started topic. I don't want to show many ways to do the one thing here.

Reply:
Makes sense, removed the explicit section. For the workflow, do you want me to do the "resolve conversation" or is that something you prefer to track yourself?